libr/asm/p/asm_pyc.c

```diff
@@ -0,0 +1,64 @@
+/* radare - LGPL3 - Copyright 2016-2020 - c0riolis, x0urc3 */
+
+#include <r_types.h>
+#include <r_lib.h>
+#include <r_util.h>
+#include <r_asm.h>
+
+#include "../arch/pyc/pyc_dis.h"
+#include "../arch/pyc/opcode.h"
+
+static pyc_opcodes *opcodes_cache = NULL;
+
+static int disassemble (RAsm *a, RAsmOp *opstruct, const ut8 *buf, int len) {
+	RList *interned_table = NULL;
+	RList *shared = NULL;
+	RList *cobjs = NULL;
+
+	RBin *bin = a->binb.bin;
+	ut64 pc = a->pc;
+
+	RBinPlugin *plugin = bin && bin->cur && bin->cur->o ? bin->cur->o->plugin : NULL;
+
+	if (plugin) {
+		if (!strcmp (plugin->name, "pyc")) {
+			shared = bin->cur->o->bin_obj;
+		}
+	}
+	cobjs = r_list_get_n (shared, 0);
+	interned_table = r_list_get_n (shared, 1);
```

Comment:
Define and assign in the same line. Move down the var definitions of the function start